.github/workflows/ci.yml

```diff
@@ -1,98 +1,29 @@
 name: CI
-on: [push, pull_request]
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
 env:
   RAILS_ENV: test
   DATABASE_URL: postgresql://postgres:@localhost/test
   DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: true
+
 jobs:
   rubocop:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Setup Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: 2.7
-      - name: Setup Bundler
-        run: |
-          gem install bundler
-          bundle config path vendor/bundle
-          bundle lock --update
-      - name: Cache gems
-        uses: actions/cache@v3
-        with:
-          path: vendor/bundle
-          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-gems-
-      - name: Setup
-        run: bundle install --jobs=3 --retry=3
```

Comment:
We don't have this steps implemented yet here: https://github.com/theforeman/actions/blob/v0/.github/workflows/rubocop.yml

Reply:
I don't think we need it. We use `ruby/setup-ruby@v1` with `bundler-cache: true` which runs `bundle install` and caches installed gems automatically for us. The `--jobs 3` and `--retry 3` is something we can ignore, IMHO.

Reply:
The jobs parameter is already set, even to 4 instead of 3: https://github.com/ruby/setup-ruby/blob/5daca165445f0ae10478593083f72ca2625e241d/bundler.js#L209
And retry defaults to 3: https://bundler.io/v2.4/man/bundle-config.1.html has
> retry (BUNDLE_RETRY): The number of times to retry failed network requests. Defaults to 3.